docker-compose/README.md

```diff
@@ -2,59 +2,66 @@
 
 This directory provides two ways to start the JSON API and Stargate coordinator with DSE 6.8 using `docker compose`.
 
-## Stargate JSON API with 3-node DSE 6.8 cluster
+## Prerequisites
+
+### Docker / Docker Compose Versions
+
+Make sure that you have Docker engine 20.x installed, which should include Docker compose 2.x. Our compose files rely on features only available in the Docker compose v2 file format.
+
+### Building the local Docker image
+If you want to use a locally built version of the JSON API Docker image rather than pulling a released version from Docker Hub, run the following command at the root of the repository to build the image:
+
+```bash
+./mvnw clean package -Dquarkus.container-image.build=true -DskipTests
+```
 
-> **PREREQUISITES:** Please build the latest JSONAPI docker image first by running: 
-> 
-> ```bash
-> ./mvnw clean package -Dquarkus.container-image.build=true -DskipTests
-> ```
-> 
-> You can control the platform using the `-Dquarkus.docker.buildx.platform=linux/amd64,linux/arm64` property.
+You can control the platform using the `-Dquarkus.docker.buildx.platform=linux/amd64,linux/arm64` property.
+
+Follow instructions under the [Script options](#script-options) section to use the locally built image.
+
+## Stargate JSON API with 3-node DSE 6.8 cluster
 
 You can start a simple Stargate configuration with the following command:
 
 ```
-docker-compose up -d
+./start_dse_68.sh
```

Comment:
rewrote instructions to prefer the wrapper script so users get the Docker version check.